### ISSUE TITLE
Break up intake-xarray into separate drivers?

### ISSUE THREAD

**Issue by danielballan** [intake/intake#469]
Intake-xarray lumps together several different I/O backends. Grouping drivers by what they _return_ (xarray in this case) is different from the usual pattern. We propose to break intake-xarray into intake-netcdf, intake-tiff with rasterio support (and maybe also tifffile support?), intake-png, and intake-zarr. We can maintain intake-xarray going forward as a metapackage that depends on these as a convenience and for back-compat.

**Comment by martindurant** [comment]
You may well have a point, but the original rationale was: that all these *call* xarray loader function

**Comment by danielballan** [comment]
Yes, I think that is sensible early on, just as we currently package a bunch of unrelated drivers together in `databroker._drivers` with the intention of splitting them up once things stabilize. Maybe this is worth doing _eventually_, with some shared utility library containing the xarray loader.
